nautobot/dcim/models/cables.py

```diff
@@ -405,6 +407,14 @@ def from_origin(cls, origin):
                     # No corresponding FrontPort found for the RearPort
                     break
 
+            elif isinstance(peer_termination, CircuitTermination):
```

Comment:
Can you please comment on this block on what it is doing and why it was needed to address this issue? It's not immediately apparent. Doesn't need to be super verbose but something for future maintainers would be helpful! :)